vendor/modules.txt

```diff
@@ -6,7 +6,7 @@ bazil.org/fuse/fuseutil
 cloud.google.com/go/compute/metadata
 # github.com/Azure/azure-pipeline-go v0.0.0-20180607212504-7571e8eb0876
 github.com/Azure/azure-pipeline-go/pipeline
-# github.com/Azure/azure-storage-blob-go v0.0.0-20180712005634-eaae161d9d5e
+# github.com/Azure/azure-storage-blob-go v0.0.0-20180906215025-bb46532f68b7
```

Comment:
I see there is an official 0.2.0 release: https://github.com/Azure/azure-storage-blob-go/releases released 15 hours ago!

Do you think it is worth updating to that?

Reply:
Agree, that was my intention, will pull the right version.